content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -283,19 +283,21 @@ Policies:
 Pods in `Pending`, `Succeeded` or `Failed` phase are always considered for eviction.
 {{< /note >}}
 
-## Arbitrary Controllers and Selectors
+## Arbitrary workloads and arbitrary selectors {#arbitrary-controllers-and-selectors}
 
 You can skip this section if you only use PDBs with the built-in
-application controllers (Deployment, ReplicationController, ReplicaSet, and StatefulSet),
-with the PDB selector matching the controller's selector.
+workload resources (Deployment, ReplicaSet, StatefulSet and ReplicationController)
+or with {{< glossary_tooltip term_id="CustomResourceDefinition" text="custom resources" >}}
+that implement a `scale` [subresource](/docs/concepts/extend-kubernetes/api-extension/custom-resources/#advanced-features-and-flexibility),
+and where the PDB selector exactly matches the selector of the Pod's owning resource.
 
-You can use a PDB with pods controlled by another type of controller, by an
+You can use a PDB with pods controlled by another resource, by an
 "operator", or bare pods, but with these restrictions:
 
 - only `.spec.minAvailable` can be used, not `.spec.maxUnavailable`.
 - only an integer value can be used with `.spec.minAvailable`, not a percentage.
```

Comment:
Nit, can we explain why in more details?

Reply:
@soltysh added the explanation for the restrictions